src/resources/ApiKeys/ApiKeysInterfaces.ts

```diff
@@ -10,7 +10,7 @@ export interface ApiKeyModel extends GranularResource {
      *
      * **Example:** t4hk287bfj5sg6wskg64ckk5a
      */
-    id?: string;
+    id: string;
```

Comment:
The `Platform.apiKeys.create` method's argument type will probably have to be updated so that it does not require the model to have an id.